### DISCUSSION TITLE
Page not scrolling on android browser after tap on patch

### DISCUSSION THREAD

**Discussion by stuartcupit** [cables-gl/cables_docs#362] [! Welcome - readme first!]
Hi, Does anyone know why once a patch has been taped on it will no longer allow the page to scroll properly? 

This only happens after the patch has been taped once. Before tapping the browser page scrolls up and down fine, after tapping the patch, no more scrolling the page just seems to bounce back to keep the patch in view. 

This only seems to happen on android, not on desktop, it even happens on an empty project on the cables site, see this link for an example https://cables.gl/p/633bf038a1293e86e06c8b73?s=5WsqE5M6B3G0pfqS or you can easily make your own to test it. 

Attached is a video showing the problem. 

https://user-images.githubusercontent.com/61870778/193891962-e0af3fa0-96c5-4722-97c1-c1c6d75aefc5.mp4

This is stopping me from integrating patches into a website as the mobile version can't scroll past the patch once it's been tapped. 

Any help greatly appreciated. Thanks.

**Comment by pandrr** [comment]
its because the cables.gl website works this way, its also on desktop, if you use mousewheel over the canvas the page is not scrolled, because this is how we expect it to work

if you want different behaviour you have to export the patch and embed it manually into your site

**Reply by stuartcupit** [reply]
Thanks for the reply. So I've exported the patch and integrated it into the site here https://solarflarestudio.co.uk/labs-test/ 
On the cables site using the patch viewer, tapping off the patch restores scrolling. But on my site, tapping off the patch doesn't seem to restore the ability to scroll on android. 
Any ideas what I'm doing wrong? 
Many thanks.

**Comment by pandrr** [comment]
are you using orbitcontrols ? this one is designed to rotate the view and disable scrolling.
if you want different behaviour you have to rotate it yourself

**Reply by stuartcupit** [reply]
Hi, It's restoring the ability to scroll the browser page after the patch has been tapped that doesn't seem to be working. I have exported and integrated the patch here  https://solarflarestudio.co.uk/labs-test/ 
The page scrolls fine until the patch is touched then you can not tap off and restore scrolling. Any ideas on how I re-enable the scrolling of the webpage when not touching the patch? 
Any help much appreciated, thanks.

**Reply by pandrr** [reply]
need to see the patch, please invite me
